src/shim.c

```diff
@@ -955,6 +968,71 @@ connect_to_proxy(struct cc_shim *shim)
 }
 
 /*
```

Comment:
nitpick: @jodh-intel do these need to be `\*!` comment starts in order to get the marked up comment notionally processed?

Reply:
Yep - good catch.

Aside: I just ran doxygen and found that there are a couple of errors that need correcting for `master` too [ #82]).